core/server/api_container/server/startosis_engine/kurtosis_instruction/run_sh/run_sh.go

```diff
@@ -332,13 +364,51 @@ func getCommandToRun(builtin *RunShCapabilities) (string, error) {
 	return commandWithNoNewLines, nil
 }
 
+func executeWithWait(ctx context.Context, builtin *RunShCapabilities, commandToRun []string) (*exec_result.ExecResult, error) {
+	// Wait is set to None
+	if builtin.wait == DisableWaitTimeoutDurationStr {
+		return builtin.serviceNetwork.RunExec(ctx, builtin.name, commandToRun)
+	}
+
+	resultChan := make(chan *exec_result.ExecResult, 1)
+	errChan := make(chan error, 1)
+
+	timoutStr := builtin.wait
+	parsedTimeout, parseErr := time.ParseDuration(timoutStr)
```

Comment:
This is extra pre-caution; i could remove it.. because I already do the validation check for wait..
```
Validator: func(value starlark.Value) *startosis_errors.InterpretationError {
						return builtin_argument.DurationOrNone(value, WaitArgName)
					},
```
@laurentluce

Reply:
If it cannot be invalid then let's remove the second check.